pkg/operator/csi/credentialsrequestcontroller/credentials_request_controller.go

```diff
@@ -41,6 +47,7 @@ func NewCredentialsRequestController(
 	manifest []byte,
 	dynamicClient dynamic.Interface,
 	operatorClient v1helpers.OperatorClientWithFinalizers,
+	operatorLister operatorv1lister.CloudCredentialLister,
```

Comment:
You'll want to wait for the informer to sync and possibly fire on state changes.